src/lib.rs

```diff
@@ -126,7 +122,7 @@ fn spawn_text(mut commands: Commands, asset_server: Res<AssetServer>) {
         .spawn_bundle(TextBundle {
             text: Text {
                 sections: vec![TextSection {
-                    value: STRING_INITIAL.to_string(),
+                    value: String::new(),
```

Comment:
If someone wanted to update the FPS every 10s, better there is some initial text, I think. Is this an issue for you?

Reply:
With `update_now`, the FPS shows one frame after the game starts. So depending on the decision on your previous comment, I'll add this back.